discord/ext/commands/converter.py

```diff
@@ -872,6 +874,14 @@ async def convert(self, ctx: Context, argument: str) -> discord.PartialEmoji:
                 id=emoji_id,
             )
 
+        if argument in EMOJIS_MAP.values():
```

Comment:
This is inefficient because `EMOJIS_MAP` is huge and only keys are hashed. Find another way

Reply:
@Lumabots you should move the EMOJIS_MAP to somwehrere where it can be reused in other places, and then create a set UNICODE_EMOJIS = set(EMOJIS_MAP.values()) and use that.